_posts/en/newsletters/2019-10-23-newsletter.md

```diff
@@ -0,0 +1,127 @@
+    into the Bitcoin consensus rules.  Adding new consensus rules to
+    Bitcoin is something that should be done carefully---because it can't
+    be undone safely for as long as anyone's bitcoins depend on those rules---so
+    it's in every user's interest
+    that a large number of technical reviewers examine the proposals for
+    possible flaws before they are implemented and before users are
+    asked to consider upgrading their full nodes to enforce the new
+    rules.  Whether through the review club or
+    in some other way, Optech strongly encourages all technically
+    skilled Bitcoin users to dedicate time to reviewing the taproot set
+    of proposals.
+
+    Anyone wanting to participate should [RSVP][trc rsvp] soon so that the
+    organizers can estimate the total number of participants and start
+    forming study groups.  To register or learn more, please see
+    the [Taproot Review Club][trc] repository.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin wallets
+and services.*
+
+- **Electrum Lightning support:** In a [series of commits this
+  month][electrum commits], Electrum has merged support for Lightning
+  Network into master. A presentation deck titled [Lightning Implementation in
+  Electrum][electrum lightning presentation] by Thomas Voegtlin provides some
+  background information and screenshots.
+
+- **Blockstream Green Tor support:** Version 3.2.4 of the Blockstream Green
+  Wallet [adds built-in Tor support][blockstream green tor article] for both iOS and Android.
+  While Tor was supported in previous Android versions, it required a separate
+  application whereas both Android and iOS versions are now bundled with Tor
+  support.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #3150][] adds new `signmessage` and `checkmessage` RPCs.
+  The first RPC will sign a message that can be verified by someone with
+  your LN node's public key.  The second RPC will verify a signed message
+  from another node either using a pubkey provided by the user or by
+  confirming the message is signed by a pubkey belonging to any known LN
+  node (e.g. a node in the set returned by the `listnodes` RPC).
+
+- [LND #3595][] raises the default maximum CLTV expiry from 1,008 blocks
+  (about 1 week) to 2,016 blocks (about two weeks).  This is the maximum
+  amount of time a new payment can be stuck as pending before it can be reclaimed
+  by its spender.  LND
+  recently tried to keep this at 1,008 blocks by decreasing the CLTV
+  delta---the minimum number of blocks each routing node along the payment path
+  has to claim its particular payment---from 144 blocks to 40 blocks
```

Comment:
"the minimum number of blocks each routing node along the payment path has to claim its particular payment" -> I think this sentence could be clearer, in the vein of replacing "has" with a better verb or reconstructing the sentence a bit.